packages/core/src/Configs/MintV3PoolConfig.class.ts

```diff
@@ -64,8 +64,15 @@ export class MintV3PoolConfig extends Config<IMintV3PoolConfigArgs> {
     return this;
   }
 
-  setFee(fee: bigint) {
-    this.fee = fee;
+  setFees(fees: bigint | IFeesConfig) {
+    this.fees =
+      typeof fees === "bigint"
+        ? {
+            ask: fees,
+            bid: fees,
+          }
+        : fees;
+
```

Comment:
This just allows the user to set the fee as a `bigint` (like now) or as an object. It will always resolve as an object from `.buildArgs`.